src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -575,24 +579,37 @@ void ov::npuw::LLMCompiledModel::export_model(std::ostream& stream) const {
     // Write config
     write(stream, m_cfg);
 
+    // Identify either full flow or weightless
+    bool is_weightless = false;
+    if (m_non_llm_props.count(ov::cache_mode.name()) &&
+        m_non_llm_props.at(ov::cache_mode.name()).as<std::string>() == "OPTIMIZE_SIZE") {
```

Comment:
```suggestion
        m_non_llm_props.at(ov::cache_mode.name()).as<CacheMode>() == CacheMode::OPTIMIZE_SIZE) {
```
BTW, weightless cache  is used for GPU in CacheMode::OPTIMIZE_SIZE, because in generic case it will drop performance of import_model.
Is it the same for NPU? or you can use weightless cache in all cases?

Reply:
It's the same in our case. With weightless flow the performance is worse

Reply:
Fixed the code

Reply:
Actually the performance is the same, just import is higher. Yet we want to keep both options for the user